onnxruntime/core/providers/js/operators/conv.h

```diff
@@ -67,7 +67,7 @@ class Conv : public JsKernel {
                                  static_cast<int32_t>(conv_attrs_.strides.size() > 0 ? conv_attrs_.strides[0] : 0),
                                  static_cast<int32_t>(conv_attrs_.strides.size() > 1 ? conv_attrs_.strides[1] : 0),
                                  static_cast<int32_t>(channels_last),
-                                 reinterpret_cast<int32_t>(&w_is_const_));
+                                 reinterpret_cast<uintptr_t>(&w_is_const_));
```

Comment:
Here and in other places we need to wrap the value in `Number` because a pointer will be passed as a BigInt and it cannot be used to index array or in any other ops (compare, add) with Numbers